src/test/javascript/spec/component/iris/ui/course-chatbot.component.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { CourseChatbotComponent } from 'app/iris/course-chatbot/course-chatbot.component';
+
+describe('CourseChatbotComponent', () => {
+    let component: CourseChatbotComponent;
+    let fixture: ComponentFixture<CourseChatbotComponent>;
+
+    beforeEach(async () => {
+        await TestBed.configureTestingModule({
+            imports: [CourseChatbotComponent],
+        }).compileComponents();
+
+        fixture = TestBed.createComponent(CourseChatbotComponent);
+        component = fixture.componentInstance;
+        fixture.detectChanges();
+    });
+
+    it('should create', () => {
+        expect(component).toBeTruthy();
+    });
+});
```

Comment:
What is the benefit of this test? 

I think that we should either add actual tests or get rid of this file

Reply:
yes we will add actual tests, for now its a barebone and has no purpose